src/eradiate/scenes/atmosphere/_core.py

```diff
@@ -372,6 +372,21 @@ class AbstractHeterogeneousAtmosphere(Atmosphere, ABC):
         init_type="float, optional",
     )
 
+    force_majorant: bool = documented(
+        attrs.field(
+            default=False,
+            kw_only=True,
+            converter=attrs.converters.optional(bool),
+            validator=attrs.validators.optional(attrs.validators.instance_of(bool)),
+        ),
+        doc="""If set to true, uses heterogeneous medium, which is compatible with all
+        integrators except PiecewiseVolpathIntegrator. Otherwise, uses a piecewise medium which 
+        is compatible with PiecewiseVolPathIntegrator and other integrators. This setting 
+        only affects PlaneParallelGeometry, other geometries use heterogeneous mediums.""",
```

Comment:
Unfortunately, our docstring formatting is rather fragile and will break with triple-quoted strings. Instead use single-quoted strings and chain them carefully:
```suggestion
        doc="If set to ``True``, use a generic heterogeneous medium implementation, compatible with all "
        "integrators except :class:`.PiecewiseVolpathIntegrator`. "
        "Otherwise, use a piecewise heterogeneous medium which "
        "is compatible with :class:`.PiecewiseVolPathIntegrator` and other integrators. "
        "This setting only applies when using a :class:`.PlaneParallelGeometry`; "
        "other geometries use the generic heterogeneous medium implementation.",
```